from cogs import error


async def main(message, message_text, numeration_command):
	usage = await error.check_errors(message, message_text, {
		"block_arguments": True,
		"commands": numeration_command
	})

	if usage[0] == 1:
		await message.reply("‚úÖ –≠—Ç—É –∫–æ–º–∞–Ω–¥—É –±–æ–ª—å—à–µ –Ω–µ –Ω—É–∂–Ω–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞—Ç—å. –î–ª—è —É–ø—Ä–æ—â–µ–Ω–∏—è, —Å–º—ã—Å–ª —Ä—É–ª–µ—Ç–∫–∏ –±—ã–ª —á—É—Ç—å –∏–∑–º–µ–Ω—ë–Ω. üí¨ –¢–µ–ø–µ—Ä—å —Ç—ã –º–æ–∂–µ—à—å —Å—Ä–∞–∑—É –¥–µ–ª–∞—Ç—å –ª—é–±—É—é —Å—Ç–∞–≤–∫—É –∏ –∫—Ä—É—Ç–∏—Ç—å —Ä—É–ª–µ—Ç–∫—É –±–µ–∑ –µ—ë –∑–∞–ø—É—Å–∫–∞", parse_mode="HTML")
	else:
		await error.send_errors(message, usage)
